Source/WebCore/platform/Length.cpp

```diff
@@ -133,7 +133,7 @@ class CalculationValueMap {
 private:
     struct Entry {
         uint64_t referenceCountMinusOne;
-        CalculationValue* value;
+        RefPtr<CalculationValue> value;
```

Comment:
If we do this, it seems we need to update `insert()` / `deref()` since they currently leakRef to make sure the pointer stays valid and adopt to delete. Seems we wouldn't need this logic anymore if we just ref in the Entry.